checkov/common/util/tqdm_utils.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import os
+import sys
+
+from colorama import Fore, Back
+from tqdm import tqdm  # type: ignore
+
+DEFAULT_BAR_FORMAT = '{l_bar}%s{bar:20}%s|[{n_fmt}/{total_fmt}]{postfix}' % (Fore.WHITE, Fore.RESET)
+SLOW_RUNNER_BAR_FORMAT = '{l_bar}%s{bar:20}%s|[{n_fmt}/{total_fmt}] %s[Slow Runner Warning]%s{postfix}' %\
+                         (Fore.LIGHTBLACK_EX, Fore.RESET, Back.YELLOW, Back.RESET)
+
+SLOW_RUNNERS = {'sca_package', 'terraform', 'cloudformation', 'helm', 'kubernetes', 'kustomize', 'secrets'}
+LOGS_ENABLED = os.environ.get('LOG_LEVEL', False)
```

Comment:
```suggestion
LOGS_ENABLED = os.getenv('LOG_LEVEL', False)
```
that's the typical one to use for such a case